src/tests/unit/tests/electron/views/device-connect-view/components/device-connect-port-entry.test.tsx

```diff
@@ -102,5 +104,25 @@ describe('DeviceConnectPortEntryTest', () => {
 
             deviceConnectActionCreatorMock.verifyAll();
         });
+
+        it('validates port through enter key', () => {
+            deviceConnectActionCreatorMock.setup(creator => creator.validatePort(testPortNumber)).verifiable(Times.once());
+
+            const props = {
+                deps: {
+                    deviceConnectActionCreator: deviceConnectActionCreatorMock.object,
+                },
+                viewState: {
+                    deviceConnectState: DeviceConnectState.Default,
+                },
+            } as DeviceConnectPortEntryProps;
+            const rendered = shallow(<DeviceConnectPortEntry {...props} />);
+            rendered.setState({ port: testPortNumber });
+            const textField = rendered.find(MaskedTextField);
+
+            textField.simulate('keydown', { keyCode: KeyCodeConstants.ENTER }, enterEventStub);
```

Comment:
do you actually need to pass `enterEventStub`? I though we figure is not needed

Reply:
You're correct; we don't need it and I've removed it in my most recent commit